README.md

```diff
@@ -33,3 +33,31 @@ The figma designs can be found [here](https://www.figma.com/file/xsPf6IIM9AevLN5
 ## Sequence diagram
 
 [![](docs/images/architecture.png)](https://docs.google.com/presentation/d/16038cTBefSKQezJk0IZKNXnSqaG2PnU07Sb2_qIkNe8/edit?usp=sharing)
+
+## User guide
+
+The first step to use the `optic-expo` application is adding a Secret. You can add it by scanning a QR code provided by the OTP Issuer or by inputing it manually.
+
+An example adding an NPM user secret:
+
+![image](https://user-images.githubusercontent.com/11404065/159178304-09a9fa80-a73c-433a-9a29-b6a48aa6af7a.png)
+
+You will see your secret added to the application home page:
+
+![image](https://user-images.githubusercontent.com/11404065/159178421-25e1f0c0-46b5-4c4a-8005-39a5fa5c4fd8.png)
+
+Finally, you need to generate your `optic-token`s by adding a new token:
+
+![image](https://user-images.githubusercontent.com/11404065/159178473-f533f308-e1a6-4240-9ad5-ce8680be0ebd.png)
+
+You can save this token to any system that need to sent to you a notification.
+The notification will ask you to approve or deny the system's read to your Secret's OTP.
+
+As you can see by the home page layout, you can generate many tokens to read the same Secret's OTP:
+
+![image](https://user-images.githubusercontent.com/11404065/159178635-418e231c-aa9c-4828-bb41-6f68cfb18059.png)
+
+By doing so, you will be able to understand who is the caller that want to read your OTP.
+
+
+The first step to use the `optic-expo` application is adding a Secret. You can add it by scanning a QR code provided by the OTP Issuer or by inputing it manually.
```

Comment:
this seems to be a copy of the first line. remove

Reply:
done